nautobot/cloud/tables.py

```diff
@@ -135,7 +136,7 @@ class CloudServiceTable(BaseTable):
     name = tables.Column(linkify=True)
     cloud_account = tables.Column(linkify=True)
     cloud_network = tables.Column(linkify=True)
-    cloud_type = tables.Column(linkify=True)
+    cloud_resource_type = tables.Column(linkify=True)
     actions = ButtonsColumn(CloudService)
```

Comment:
On looking at this again, probably need to add a `TagsColumn` here as well:

```suggestion
    tags = TagColumn(url_name="cloud:cloudservice_list")
    actions = ButtonsColumn(CloudService)
```